chapter2/tests/t13/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1.2 
+FROM --platform=${BUILDPLATFORM:-linux/amd64} node:14-slim AS builder
+
+RUN apt update
+RUN apt install git -y
+
+WORKDIR /opt/flatris
+RUN git clone https://github.com/rtest12/flatris.git /opt/flatris
+
+RUN yarn install --arch=$BUILDPLATFORM
+RUN yarn test
+RUN yarn build
+
+FROM --platform=${BUILDPLATFORM:-linux/amd64} nginx:1.22.0-alpine AS runtime
```

Comment:
Why is nginx, and not just alpine, the nginx functionality, somehow involved?

Reply:
@udaltsovra not sure  I understand. I also prefer purpose specific images to general ones. I.e. python:3.10 to alpine, nginx:x.x.x to generic alpine.
?

Reply:
@dmitry-mightydevops The alpine image weighs from 5.5 to 7 MB, the fresh nginx image even on alpine will weigh 40 MB. If there is no need for the functionality of the web server, I do not see the need to use its image. If there is a need to use nginx as a reverse proxy in this task, then I would prefer compose rather than combining several services into one image.

Reply:
@udaltsovra 
1. if nginx is not used in this task then indeed nginx base image is useless
2. alpine has DNS issue that many people complained: https://github.com/k3s-io/k3s/issues/6132, https://bell-sw.com/blog/how-to-deal-with-alpine-dns-issues/
3. multiple containers would lead to excessive use of IP addresses, exhaust the capacity of available PODs in the kubernetes. So there should be always a balance between quantity and quality (i.e. ease of maintenance)

Reply:
> Why is nginx, and not just alpine, the nginx functionality, somehow involved?

@dmitry-mightydevops 
@udaltsovra 
I suppose we'll go with the nginx option, I like it more.